test/lint/lint-includes.sh

```diff
@@ -97,4 +100,12 @@ for EXPECTED_BOOST_INCLUDE in "${EXPECTED_BOOST_INCLUDES[@]}"; do
     fi
 done
 
+QUOTE_SYNTAX_INCLUDES=$(git grep '^#include "' -- "**/*.cpp" "**/*.h" | grep -Ev "${IGNORE_REGEXP}")
```

Comment:
Why not use `*.cpp` and `*.h`?

Reply:
Yes, that would be equivalent in this specific case (since we're only matching in `src/`), but my suggestions is that we keep this PR as is since it already has two utACK:s that would need re-reviewing in case of a change. Makes sense? :-)

Reply:
Fixed when rebasing :-)